bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/protocol/TelenotMessage.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2021 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.telenot.internal.protocol;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Superclass for all Alarm Decoder protocol message types.
+ * 
+ * @author Ronny Grun - Initial contribution
+ */
+@NonNullByDefault
+public abstract class TelenotMessage {
+
+    protected static final Pattern SPLIT_REGEX = Pattern.compile("[^\\,\"]+|\"[^\"]*\"");
+
+    /** string containing the original unparsed message */
+    public final String message;
+
+    public TelenotMessage(String message2) {
+        this.message = message2;
+    }
+
+    @Override
+    public String toString() {
+        return message;
+    }
+
+    /** Utility routine to split an Telenot message into its component parts */
+    protected static List<String> splitMsg(String msg) {
```

Comment:
Why is this static? Wouldn't it be possible to let it operate on the message and use `message.split()` instead of calling `splitMsg(message)`?